map/src/data/index.ts

```diff
@@ -45,3 +47,23 @@ export const SERVICE_TYPES = Object.keys(SERVICES) as Service[];
 export interface Filter {
   service?: Service;
 }
+
+export const LANGUAGES = {
+  en: {
+    label: t(s => s.languages.en),
+  },
+  ru: {
+    label: t(s => s.languages.ru),
+  },
```

Comment:
I think rather than providing language names in the current language, it's better to display languages in the *target* language, so translations for language names are unnecessary.